src/containers/RegisterContainer/RegisterBindAccountContainer.js

```diff
@@ -1,19 +1,7 @@
 import { reduxForm, getFormValues } from "redux-form";
 import { connect } from "react-redux";
 import RegisterBindAccount from "../../components/Register/RegisterBindAccount";
-import fieldIsRequired from "../../components/helpers/validators/required-fields-validator";
-import isValidEmail from "../../components/helpers/validators/email-address-validator";
-
-// TODO Take out validationl logic
-
-const validate = values => {
-  const requiredFields = ["password", "email", "type"];
-
-  return {
-    ...fieldIsRequired(values, requiredFields),
-    ...isValidEmail(values.email),
-  };
-};
+import { validate } from "../../components/helpers/validateHelper";
 
 const mapStateToProps = state => {
```

Comment:
try to destructure state

Reply:
This is not my component .My task was to create universal vallidation function and change in all places.